packages/lab/src/list-next/ListItemNext.tsx

```diff
@@ -78,7 +87,7 @@ export const ListItemNext = forwardRef<HTMLLIElement, ListItemNextProps>(
         aria-selected={selected || undefined}
         id={itemId}
         data-value={value}
-        onClick={handleClick}
+        onClick={disabled ? undefined : handleClick}
```

Comment:
what's our go to for this? the above, or putting an if !disabled in the handleClick?

Reply:
not sure we have one, but @alycrys suggested this implementation, seems like we are already doing it in accordion too